### DISCUSSION TITLE
Mail form via HTML

### DISCUSSION THREAD

**Discussion by Rudy1952** [community/community#131967] [Actions] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Ik probeerde een Mail form aan te maken met html :
<TD WIDTH=100 style="height: 80px" class="style5"><a href="https://www.stamboomvansantvoort.be/humo-gen/index.php?page=mailform">
<IMG src=e-mail.gif width=80 border=0 ALT="E-mail mij" style="float: left" height="70"></A></TD>
(geen reactie in mijn browser Microsoft Edge of Google Chrome)
Bestaat er een mogelijkheid om een Mail form te maken via php ???

**Comment by Aditijainnn** [comment]
https://www.lcn.com/support/articles/how-to-create-an-email-form-with-php/
or 
https://github.com/johnkueh/php-form-to-email

Hope these might help you

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Rudy1952 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.

Please note for future posts: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
